ch_tools/monrun_checks_keeper/main.py

```diff
@@ -66,6 +66,7 @@ def wrapper(ctx, *a, **kw):
                 status.set_code(1)
 
             log_message = f"Completed with {status.code};{status.message}"
+            logging.disable_stdout_logger()
```

Comment:
Message with status code will be printed later in `status.report(ctx)`. Maybe we should replace `print` in `report` for logging and remove `log_status` from here.
Also, do we need to print exception info from the try block above? If not, we can disable stdout before logging exception.

Reply:
Lets leave `print`. I think we can move `disable_stdout_logger` up.